cmd/fuego/templates/controller/service.go

```diff
@@ -0,0 +1,90 @@
+package controller
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/go-fuego/fuego"
+)
+
+type NewControllerServiceImpl struct {
```

Comment:
Why `NewController` ? Kind of confusing for future contributors. You can name it `NewEntity` maybe?